packages/dependency-royalty-management/contracts/MultiRoyaltyDistributor.sol

```diff
@@ -79,6 +80,7 @@ abstract contract MultiRoyaltyDistributor is IEIP2981, IMultiRoyaltyDistributor,
             }
             royaltyConfig.tokenId = tokenId;
             royaltyConfigs[i] = royaltyConfig;
+            royaltyConfigs[i].royaltyBPS = contractRoyaltyBPS;
```

Comment:
I am a bit confused here.
This function should return as per description: `Returns royalty receivers and their split of royalty for each token`

So it returns how the royalty is split or the royalty %?
Right now you will return the contract royalty for each token so lets say `5%` but we don't return how it is split

Reply:
See https://github.com/manifoldxyz/royalty-registry-solidity/blob/main/contracts/overrides/MultiReceiverRoyaltyOverrideCore.sol
Do we need to discuss the approach here
@rishabh0x00 @wojciech-turek

Reply:
I don't think this function is used anywhere right?
Seems like it would be only useful if we had different royalties on different tokens

Reply:
@rishabh0x00 are we going to delete this function?